test/config/crds/creatures_v2alpha1_kraken.yaml

```diff
@@ -1,13 +1,33 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  creationTimestamp: null
   labels:
     controller-tools.k8s.io: "1.0"
   name: krakens.creatures.k8s.io
 spec:
   group: creatures.k8s.io
-  version: "v2alpha1"
   names:
     kind: Kraken
     plural: krakens
   scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          type: object
+        status:
+          type: object
+      type: object
+  version: v2alpha1
+status:
```

Comment:
Why those new field got generated?

Reply:
This is because we are YAML marshaling the CRD object, the previous generation was template based.